core/drivers/stm32_i2c.c

```diff
@@ -1151,6 +1152,86 @@ int stm32_i2c_master_transmit(struct i2c_handle_s *hi2c, uint32_t dev_addr,
 	return i2c_write(hi2c, &request, p_data, size);
 }
 
+/*
+ * Optimized 1 byte read/write function for unpaged sequences.
+ * 8-bit addressing mode / single byte transferred / use default I2C timeout.
+ * 'unpg88' refers to 8bit address/8bit value.
```

Comment:
`unpg88` is not used

Reply:
thanks. will remove this deprecated comment.